demonstration/linux/include/posix/libcyphal/transport/udp/session/input_session.hpp

```diff
@@ -40,7 +44,11 @@ class PosixMessageSubscriber final : public MessageSubscriber
     PosixMessageSubscriber(const NodeID node_id, const ip::v4::Address local_address) noexcept
```

Comment:
FYI, I changed the name of this class from `PosixMessageSubscriber` to `PosixInputSession` because it is going to be used for receiving both message and service transfers for both UDP and CAN. 

This name change is more in line with the name of this file (`input_session.hpp`) and with PyCyphal. 

Same for `PosixMessagePublisher`, renamed to `PosixOutputSession`